ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po

```diff
@@ -3009,6 +3015,11 @@ msgstr ""
 msgid "Skip to site information"
 msgstr ""
 
+#: ckanext/canada/templates/public/user/confirm_delete.html:8
+#, fuzzy
```

Comment:
@RabiaSajjad need to update this. Sometimes the compile catalog does this. Just removing what is in the `msgstr` and remove the fuzzy should do